internals/overlord/servstate/manager.go

```diff
@@ -548,7 +548,13 @@ func (m *ServiceManager) SetServiceArgs(serviceArgs map[string][]string) error {
 	return m.appendLayer(newLayer)
 }
 
-// servicesToStop returns a slice of service names to stop, in dependency order.
+// servicesToStop is used during service manager shutdown to cleanly terminate
```

Comment:
This is a great comment, however, it now doesn't say what the return value means (a slice of service names to stop, in dependency order). Maybe prefix the new comment with the old one?